packages/flutter/test_fixes/material.dart.expect

```diff
@@ -538,4 +538,254 @@ void main() {
   ButtonStyle textButtonStyle = TextButton.styleFrom(
     foregroundColor: Colors.blue, disabledForegroundColor: Colors.grey.withOpacity(0.38),
   );
+
+  // Changes made in https://github.com/flutter/flutter/pull/97972
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(checkboxTheme: CheckboxThemeData(
+ 	  fillColor: MaterialStateProperty.resolveWith<Color?>((Set<MaterialState> states) {
+      if (states.contains(MaterialState.disabled)) {
+        return null;
+      }
+      if (states.contains(MaterialState.selected)) {
+        return Colors.black;
+      }
+ 	    return null;
+ 	  }),
+ 	  )radioTheme: RadioThemeData(
+ 	   fillColor: MaterialStateProperty.resolveWith<Color?>((Set<MaterialState> states) {
+       if (states.contains(MaterialState.disabled)) {
+         return null;
+       }
+       if (states.contains(MaterialState.selected)) {
+         return Colors.black;
+       }
+ 	     return null;
+ 	   }),
+ 	  )switchTheme: SwitchThemeData(
```

Comment:
Several of your expectations here are missing commas now.. why?

```suggestion
 	  ), switchTheme: SwitchThemeData(
```

Reply:
In case this was just a wrong-file change, the commas should removed from the fix_data.yaml. Not the .expect files.

Reply:
> In case this was just a wrong-file change, the commas should removed from the fix_data.yaml. Not the .expect files.

The test fails as .expect doesn't generate commas on my machine after updating `fix_data.yaml`.

![Screenshot 2022-08-02 at 19 56 44](https://user-images.githubusercontent.com/48603081/182431665-7befa632-53ec-417c-912e-ccea40a840ef.png)

Reply:
I also deleted `bin/cache` folder before testing.

Reply:
I am updating this again on a different machine, I can see commas are generated again. Apologies, I have to reset my main machine.